src/test/javascript/spec/component/exam/manage/exam-exercise-import.component.spec.ts

```diff
@@ -299,4 +299,123 @@ describe('Exam Exercise Import Component', () => {
         expect(component.getExerciseIcon(quizExercise)).toEqual(faCheckDouble);
         expect(component.getExerciseIcon(fileUploadExercise)).toEqual(faFileUpload);
     });
+
+    describe(`Programming exercise import validation`, () => {
+        const exerciseGroup6 = { title: 'exerciseGroup6' } as ExerciseGroup;
+        const programmingExercise2 = new ProgrammingExercise(undefined, exerciseGroup6);
+
+        programmingExercise2.id = 6;
+        programmingExercise2.title = 'ProgrammingExercise';
+        programmingExercise2.shortName = 'progEx1';
+
+        const programmingExercise3 = new ProgrammingExercise(undefined, exerciseGroup6);
+        programmingExercise3.id = 7;
+        programmingExercise3.title = 'ProgrammingExercise';
+        programmingExercise3.shortName = 'progEx2';
+        exerciseGroup6.exercises = [programmingExercise2, programmingExercise3];
+
+        const exam2 = {
+            id: 10,
+            exerciseGroups: [exerciseGroup1, exerciseGroup2, exerciseGroup3, exerciseGroup4, exerciseGroup5, exerciseGroup6],
+        } as Exam;
+
+        beforeEach(() => {
+            component.exam = exam2;
+            component.importInSameCourse = false;
+            component.ngOnInit();
+        });
+
+        afterEach(() => {
+            programmingExercise2.title = 'ProgrammingExercise';
+            programmingExercise2.shortName = 'progEx1';
+            programmingExercise3.title = 'ProgrammingExercise';
+            programmingExercise3.shortName = 'progEx2';
+        });
+
+        it('should remove duplicated titles when importing', () => {
+            expect(programmingExercise.title).toBe('ProgrammingExercise');
+            expect(programmingExercise2.title).toBe('');
+            expect(programmingExercise3.title).toBe('');
+        });
+
+        it.each([true, false])('should check for programming exercise duplicated titles and short names when entering input', (duplicatedTitles) => {
+            const duplicatesToCheck = duplicatedTitles ? component.exercisesWithDuplicatedTitles : component.exercisesWithDuplicatedShortNames;
```

Comment:
Can you use a string key here so that it is immediately visible what case fails from the logs? Something like:
```suggestion
        it.each(["exercisesWithDuplicatedTitles", "exercisesWithDuplicatedShortNames"])('should check for programming exercise duplicated titles and short names when entering input', (attrToCheck) => {
            const duplicatesToCheck = component[attrToCheck];
```

Reply:
That's a really good idea, thanks!